.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "singleQuote": true,
```

Comment:
Sorry to actually bike shed on this. 🙈

For JS, I'm usually team single quote, but our code base seems to use mostly double quotes. If it's okay with everyone, I think we should probably use double quotes to make this less disruptive.